IGCexpansion/CodonGeneconv.py

```diff
@@ -176,17 +182,23 @@ def get_initial_x_process(self, transformation = 'log'):
 
         if self.Model == 'MG94':
             # x_process[] = %AG, %A, %C, kappa, omega, tau
-            if not self.use_IGC_Omega():
+            if not (self.use_IGC_Omega() or self.use_Homo_Omega()):
                 self.x_process = np.log(np.array([count[0] + count[2], count[0] / (count[0] + count[2]), count[1] / (count[1] + count[3]),
                                   self.kappa, self.omega, self.tau]))
             elif self.IGC_Omega is not None:
                 self.x_process = np.log(
                     np.array([count[0] + count[2], count[0] / (count[0] + count[2]), count[1] / (count[1] + count[3]),
                               self.kappa, self.omega, self.IGC_Omega, self.tau]))
-            else:
+            elif self.Tau_Omega is not None:

```

Comment:
combine these two conditions into `elif self.use_IGC_Omega(): if self.IGC_Omega is not None:... elif self.Tau_Omega is not None:...`